src/components/User/ContestsContainer.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div>
+    <div :class="$style.title">実績</div>
+    <p v-for="contest in contests" :key="contest.id">{{ contest.name }}</p>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent, PropType } from 'vue'
+import { ContestTeamWithContestName } from '/@/lib/apis'
+
+export default defineComponent({
+  name: 'ContestsContainer',
+  props: {
+    contests: {
+      type: Object as PropType<Array<ContestTeamWithContestName>>,
```

Comment:
```suggestion
      type: Array as PropType<ContestTeamWithContestName[]>,
```
`Object`じゃなくて`Array`ですね
あと、複雑な型じゃないなら`[]`を使ったほうがよさそうです